models/admission.py

```diff
@@ -408,6 +408,7 @@ class Admission(Model):
 
     condition_of_acceptance = models.TextField(
         blank=True,
+        null=True,
```

Comment:
Est-ce que c'est nécessaire ? Il me semble que pour in TextField on peut stocker une chaine vide, et que ce n'est pas équivalent à null... à vérifier ?